### PR TITLE
Added Browser Env

### PR DESCRIPTION
**Pull request by maZahaca**

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+cp.spawnSync('babel', ['lib', '-d', 'build/lib'], { stdio: 'inherit' });
+delete pkg.private;
+delete pkg.devDependencies;
+delete pkg.scripts;
+delete pkg['pre-commit'];
+delete pkg['lint-staged'];
+fs.writeFileSync('build/package.json', JSON.stringify(pkg, null, '  '), 'utf-8');
+fs.writeFileSync('build/LICENSE', fs.readFileSync('LICENSE', 'utf-8'), 'utf-8');
+fs.writeFileSync('build/README.md', fs.readFileSync('README.md', 'utf-8'), 'utf-8');
+cp.spawnSync('cp', ['-R', 'vendor', 'build/vendor']);
```
Comment (jifeon) [inline build.js]: don't you think vendors should live in abstract env?

File: lib/BrowserEnvironment.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    debug('Initializing..');
+    super(options);
+  }
+
+  evaluateJs() {
+    const args = Array.prototype.slice.call(arguments, 0);
+
+    const evalFunc = args.pop();
+    if (typeof evalFunc !== 'function') {
+      throw new Error('You must pass function as last argument to PhantomEnvironment.evaluateJs');
```
Comment (jifeon) [inline lib/BrowserEnvironment.js]: PhantomEnvironment -> BrowserEnvironment

File: build.js (hunk 1 above)
Comment (maZahaca) [reply build.js]: You're right!

**Comment by maZahaca** [comment]
@jifeon have a look once again
Сan you explain how can I inject vendors in browser env?

**Review by jifeon — APPROVED** [review]
Looks great!

**Comment by jifeon** [comment]
> Сan you explain how can I inject vendors in browser env?

How did you do it before? With new abstract env, you can redefine `_injectFiles` method.

**Comment by maZahaca** [comment]
@jifeon that's the thing. It wasn't there before. But I has to...

**Pull request merged**